docs/languages/PHP.md

```diff
@@ -67,6 +67,15 @@
     }
     ```
 
+1. Class usages MUST be in alphabetical order.
```

Comment:
Should this be SHOULD since there are existing files when this not the case? Not sure if this should affect future decisions although it does mean we couldn't add a rule to the code sniffer.

Reply:
I'd prefer it to be a MUST if we're going to enforce it, but appreciate it's not something we can add to sniff easily seeing as there are multiple classes where they're not in alphabetical order. I've updated it to a SHOULD, but might be something we can turn into a MUST later on down the line.